go.mod

```diff
@@ -1,318 +1,322 @@
 module github.com/ionos-cloud/octopinger
 
-go 1.22.4
+go 1.23.7
 
 require (
 	github.com/caarlos0/env/v6 v6.10.1
-	github.com/chenjiandongx/pinger v0.2.0
-	github.com/go-logr/logr v1.2.4
+	github.com/chenjiandongx/pinger v0.3.0
+	github.com/go-logr/logr v1.4.2
 	github.com/gofiber/adaptor/v2 v2.2.1
```

Comment:
please remove this in favor of https://github.com/gofiber/fiber/tree/v2/middleware/adaptor